src/main/webapp/app/shared/layouts/navbar/navbar.component.ts

```diff
@@ -544,6 +544,11 @@ export class NavbarComponent implements OnInit, OnDestroy {
                     this.addTranslationAsCrumb(currentPath, segment);
                 }
                 break;
+            case 'submissions':
+                // only a scores list exists, no special one for submissions
+                const updatedLink = currentPath.replace('/submissions/', '/scores/');
+                this.addTranslationAsCrumb(updatedLink, 'submissions');
```

Comment:
I think keeping submissions makes sense, since I’m assessing a student’s submission to the exercise as a tutor, not their score.

Reply:
Also true, I was only thinking from the perspective of available pages. When we consider the path to this page we could also link back to the assessment dashboard instead of the scores page, but that's out of scope for this bugfix.